.github/workflows/pre-commit.yaml

```diff
@@ -56,10 +56,18 @@ jobs:
 
     - name: Get current date
       id: date
-      run: echo "::set-output name=date::$(date '+%d-%m-%Y %H:%M:%S')"
+      run: echo "date=$(date '+%d-%m-%Y %H:%M:%S')" >> $GITHUB_OUTPUT
 
     - name: Run pre-commit checks
-      run: while ! pre-commit run -a ; do pre-commit run -a ; done || exit 0
+      run: |
```

Comment:
There is a simpler way with a for loop, ex `for run in {1..10}; do command; done`

Reply:
I have added this @hugoalmeida264 and tested successfully. Please have another look